data/json/professions.json

```
@@ -537,6 +537,22 @@
       "female": [ "bra", "panties" ]
     }
   },
+  {
+    "type": "profession",
+    "id": "vid_blogger",
+    "name": "Video Blogger",
+    "description": "You're a video blogger, you create video content and post it on various social medias.  Now most of your fans become undead, alongside your remaining equipments, you may want to see if your social skills learnt from creating content help you somehow.",
```

```suggestion
    "description": "You're a video blogger, creating and commenting video content and then posting it on various platforms.  Now most of your fans are dead, alongside your equipment. Maybe leading a fanbase, moderating myriads of comments and surviving shitstorms has taught you something valuable after all.",
```

Some rewording. I feel like the original description is, no offense meant, a bit clumsy. Mine could use some improvement, too but you should see what I tried to improve.

I used "various platforms" to circumvent the awkward "social medias" mention, even though merriam webster said its both singular and plural.